### PR TITLE
feat: add `constants/float32/min-base10-exponent-subnormal`

### LINKED ISSUE #3094: [RFC]: add `constants/float32/min-base10-exponent-subnormal`
### Description

This RFC proposes

- This RFC proposes adding the package `@stdlib/constants/float32/min-base10-exponent-subnormal` returning minimum base 10 exponent for a subnormal single-precision floating-point number.

```javascript
var FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL = require( '@stdlib/constants/float32/min-base10-exponent-subnormal' );

console.log( FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL );
// => -45
```

### Related Issues

_No response_

### Questions

No.

### Other

No.

### Checklist

- [x] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [x] Searched for existing issues and pull requests.
- [x] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by aayush0325**
Resolves #3094

## Description

> What is the purpose of this pull request?

This pull request:

-   adds `constants/float32/min-base10-exponent-subnormal`

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #3094

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

**Comment by aayush0325** [comment]
ready for review @Planeshifter !

File: lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/include/stdlib/constants/float64/min_base10_exponent_subnormal.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* limitations under the License.
+*/
+
+#ifndef STDLIB_CONSTANTS_FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL_H
+#define STDLIB_CONSTANTS_FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL_H
+
+/**
+* Macro for the minimum base 10 exponent for a subnormal single-precision floating-point number.
+*/
+#define STDLIB_CONSTANT_FLOAT64_MIN_BASE10_EXPONENT_SUBNORMAL -45
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/include/stdlib/constants/float64/min_base10_exponent_subnormal.h]: Should be `FLOAT32`. We also need to move the header file, as it currently resides in a `float64` directory.

```suggestion
#define STDLIB_CONSTANT_FLOAT32_MIN_BASE10_EXPONENT_SUBNORMAL -45
```

File: lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  ],
+  "keywords": [
+    "stdlib",
+    "stdmath",
+    "constant",
+    "const",
+    "mathematics",
+    "math",
+    "double",
+    "dbl",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/package.json]:
```suggestion
    "single",
```

**Review by Planeshifter — CHANGES_REQUESTED** [review]
Thanks! Left two comments with requested changes.

File: lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/include/stdlib/constants/float64/min_base10_exponent_subnormal.h (hunk 1 above)
Comment (aayush0325) [reply lib/node_modules/@stdlib/constants/float32/min-base10-exponent-subnormal/include/stdlib/constants/float64/min_base10_exponent_subnormal.h]: pushed the changes! thanks @Planeshifter

**Review by Planeshifter — APPROVED** [review]
Thanks for the edits! Looking good now; will merge shortly.

**Pull request merged**